src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -654,7 +658,7 @@
 examples:
   - name: Add a timer trigger to a task.
     text: >
-        az acr task timer add -n taskname -r registryname --timer-name myTimer --schedule "30 9 * * 1-5"
+        az acr task timer add -n taskname -r registryname --timer-name t2 --schedule "30 9 * * 1-5"
```

Comment:
nit: why changed to `t2`?

Reply:
The output does not wrap when printing help. Decided to shorten the name similar to the default trigger name as opposed to putting a new line